gradle-plugin/src/main/java/com/contrastsecurity/gradle/plugin/ContrastConfigurationExtension.java

```diff
@@ -0,0 +1,95 @@
+package com.contrastsecurity.gradle.plugin;
+
+/** Extension for configuring TeamServer API Credentials for downloading agent */
+public class ContrastConfigurationExtension {
+  private String username;
+  private String apiKey;
+  private String serviceKey;
+  private String apiUrl;
+  private String orgUuid;
+  private String appName;
+  private String serverName;
+  private String jarPath;
+  private String appVersion;
+  private boolean attachToTests;
+
+  public void setUsername(final String username) {
```

Comment:
None of these setters are used, but I _think_ they're necessary for groovy to make sense of this class when it's used a build script. I can double check that

Reply:
I know people hate it but we should really be using groovy to write these things...  Groovy removes the confusion about things like that.

Reply:
I'm not against it in this case